pygmt/figure.py

```diff
@@ -395,8 +395,8 @@ def shift_origin(self, xshift=None, yshift=None):
         """
         Shift plot origin in x and/or y directions.
 
-        This method shifts plot origin relative to the current origin by
-        (*xshift*, *yshift*) and optionally append the length unit (**c**,
+        This method shifts the plot origin relative to the current origin
+        by (*xshift*, *yshift*). Optionally, append the length unit (**c**,
         **i**, or **p**).
```

Comment:
What's the default unit if c/i/p is not given?

Reply:
The default is `c`.

Reply:
Maybe add a sentence like the following to be clear?
```suggestion
        by (*xshift*, *yshift*). Optionally, append the length unit (**c**,
        **i**, or **p**). Default unit if not given is **c** for centimeter.
```